pages/_app.js

```diff
@@ -45,6 +113,9 @@ function MyApp({ Component, pageProps }) {
         exercises={exercises}
         onAddWorkout={handleAddWorkout}
         workouts={workouts}
+        onDelete={handleDeleteWorkout}
```

Comment:
nur dass du später irgendwann noch weißt, was was ist.... (wenn du dich entscheidest, es umzubenennen, würde sich das dann auch auf die Unterseiten auswirken, wenn du die Funktion entgegennimmst)
```suggestion
        onDeleteWorkout={handleDeleteWorkout}
```